openehr-rm/src/main/java/com/nedap/archie/rm/datavalues/encapsulated/DvParsable.java

```diff
@@ -69,4 +89,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(super.hashCode(), value, formalism);
     }
+
+    @Invariant("Formalism_valid")
+    public boolean formalismValid() {
+        return InvariantUtil.nullOrNotEmpty(formalism);
+    }
+
+    @Invariant("Size_valid")
+    public boolean sizeValid() {
+        return size() > 0;
```

Comment:
```suggestion
        return size() >= 0;
```
according to https://specifications.openehr.org/releases/RM/latest/data_types.html#_dv_parsable_class

Reply:
that makes this not a data validation, but a correctness rule. I will change it, and set it to be ignored, since the implementation does not need to be validated during data validation.